.changelog/2825.txt

```diff
@@ -1,11 +1,11 @@
-```release-note:note
```

Comment:
pass by fix of these changelog entries so they are places within enhancement section